packages/@angular/cli/commands/test.ts

```diff
@@ -55,6 +55,11 @@ const TestCommand = Command.extend({
         Defaults to the karma config file in .angular-cli.json.`
     },
     {
+      name: 'aot',
```

Comment:
also have to add the same field into `TestOptions` interface

Reply:
Thanks for pointing out, I fixed it :)